src/services/users/users.ts

```diff
@@ -48,7 +48,9 @@ export default class UserSdk {
   }
 
   createUser(identityToken: string): Promise<CreateUserResponse> {
-    return this.axiosClient.post<CreateUserResponse>(
+    let axiosClientPostUsers = this.axiosClient
+    axiosClientPostUsers.defaults.headers.common["Authorization"] = identityToken;
```

Comment:
This is a little dangerous if you don't clean it up afterwards as it will now be used in all requests that are fired off by `this.axiosClient`. Not sure if that's relevant or not for `users.ts`

Reply:
Instead, you could just pass it in the headers of the request rather than making it a default header

Reply:
Yeah so we actually did that before and for some reason CORS got triggered anyways, it seems to be super finicky so hopefully this will work in the meantime until users is refactored.